concreate/cli.py

```diff
@@ -44,8 +46,19 @@ def run(self):
                             action='version',
                             help='show version and exit', version=version)
 
-        build_group = parser.add_argument_group(
-            'build', "Arguments valid for the 'build' target")
+        test_group = parser.add_argument_group('test',
+                                               "Arguments valid for the 'test' target")
+
+        build_group = parser.add_argument_group('build',
+                                                "Arguments valid for the 'build' target")
+
+        test_group.add_argument('--test-tags',
+                                action='append',
+                                help='tags used for behave test')
```

Comment:
What's the use case for it? Besides WIP and images to test - we offer also to define our own tags? How this will work with the AND/OR logic?

Reply:
It works same as behave and or logic - it should be mapped 1:1 if you think its not needed i can drop it.

Reply:
So, what I type in this argument is passed directly to behave? And this overrides anything, right?

Reply:
yes - it shoudl work that way, but if I'm think more about it its unneeded probably - we can achieve similar behaviour by overrides.yaml. SO I'll drop it ok?